### PR TITLE
Support for gathering Aggregate and Server Group data

### PR DESCRIPTION
**Pull request by timbyr**
Condensation will gather additional data:
Host aggregates
Server groups
Flavor extra_specs

### CODE REVIEW

**Comment by roman-verchikov** [comment]
Aggregates are unrelated to condensation step, so should be implemented as a separate command.

**Comment by roman-verchikov** [comment]
Once again, this should not be part of condensation, please move those changes out

File: fabfile.py [hunk 1]
```diff
@@ -193,6 +195,13 @@ def get_condensation_info(name_config=None):
 
 
 @task
+def get_nova_info(name_config=None):
```
Comment (roman-verchikov) [inline fabfile.py]: Please add docstring describing what this task does, how and when to use it

**Comment by roman-verchikov** [comment]
Looks good, please add docstring for the fab command.

**Comment by roman-verchikov** [comment]
@timbyr please add docstring to `fab` command introduced

**Comment by roman-verchikov** [comment]
+1, needs to be tested in staging

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.collection.test_nova
tests/collection/test_nova.py:13: [E1002(super-on-old-class), CollectTest.setUp] Use of super on an old style class
tests/collection/test_nova.py:23: [E1101(no-member), CollectTest.setUp] Instance of 'dict' has no 'Client' member

**Comment by MirantisWorkloadMobility** [comment]
Ready

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**